### DISCUSSION TITLE
NOAA data - are all sources "equal"?

### DISCUSSION THREAD

**Discussion by williamhobbs** [blaylockbk/Herbie#265] [General]
### Background:
This is related to PR https://github.com/blaylockbk/Herbie/pull/248 and issue https://github.com/blaylockbk/Herbie/issues/247. I'm having odd issues with a corporate firewall and AWS. We can now simply use Azure or Google instead, which is great, but I'm wondering if there are any downsides to giving up on getting AWS to work on our network.

### Main questions:
1. Are all the sources for, e.g., GEFS and HRRR through NODD equivalent? For example, does NOAA intend to use AWS for operational and archival purposes, while Azure and Google are just for archival purposes and therefore might have higher latency in making real-time models available? Maybe it just started being used sooner, but I feel like I see a lot more references to AWS...
2. Where would I find "official" info on this, or ask NOAA for clarification? I'm aware of https://github.com/NOAA-GSL/GSL-Regional-Model-Forum/discussions and the NCEP NOMADS Users mailing list, but neither of those seem like the right place to ask about broader approach to AWS/Azure/Google. 

### Additional comments:

I seem to recall that I've seen different lengths of historical data in the different sources, which is relevant, but I'm more interested in ongoing latency, reliability, etc. Obviously, NOAA can't control all aspects of what the 3rd parties do and how they perform, but I'm wondering if they (NOAA) have their own prioritization, or if anyone has observed consistent meaningful differences.

**Comment by cmo84** [comment]
All roads point to nodd@noaa.gov as being the official contact for this.  See https://www.noaa.gov/big-data-project-frequently-asked-questions

**Reply by williamhobbs** [reply]
Perfect. Thanks!

**Reply by Patrick-Keown** [reply]
Hi Will, 

As cmo84 mentioned, nodd@noaa.gov is the best contact for NOAA data being disseminated to AWS, Google, and Azure through the NOAA Open Data Dissemination (NODD) Program. 

With regard to the three cloud service providers we work with, we do replicate many datasets across all three cloud services providers that we work with. We do our due diligence to ensure that data is arriving on all three in the most efficient manner possible in order to provide the lowest latency. The data reliability from our delivery end, is the same across the platforms and then the storage reliability depends on that cloud service providers service level agreement with their public customers. To directly answer your first question we do not utilize one for archive and another for dissemination. All data we move to the cloud service providers is done so for public consumption and its delivery is based on the manner at which it is produced within NOAA. 

To address your additional comments, there may be different historical records on each provider. This has to do with the inter workings of our relationships with them. If you have data that you would like to see on the cloud, please let us know. 

Thanks,

Patrick Keown
NODD Program Manager
NOAA Office of the Chief Information Officer

**Reply by williamhobbs** [reply]
Thanks, @Patrick-Keown!
